### PR TITLE
Fix: failed tests related to UTType on M1 Mac 

### PR DESCRIPTION
**Pull request by billypchan**
### Issues

Some tests related to UTType resolving fail on M1 MacBook

### Causes

custom struct `UTType` not works due to calling deprecated methods.

### Solutions

Do not use custom struct `UTType` and replaced with iOS 14 save methods.

### dependency

- [x] https://github.com/wireapp/wire-ios-utilities/pull/122
- [x] fix  ZMConversationMessagesTests.testThatWeCanInsertAPassFileMessage()

### CODE REVIEW

File: Source/Model/Message/RasterImages+Protobuf.swift [hunk 1]
```diff
@@ -18,29 +18,19 @@
...
-        }
-        #else
-        guard UTType(mimeType: mimeType)?.isSVG == false else {
-            return false
-        }
-        #endif
-        return true
+        guard let uti = UTIHelper.convertToUti(mime: mimeType) else { return false }
+        
+        return !UTIHelper.conformsToVectorType(uti: uti)
```
Comment (KaterinaWire) [inline Source/Model/Message/RasterImages+Protobuf.swift:27]: Why not to use `conformsToImageType`?

Comment (billypchan) [reply Source/Model/Message/RasterImages+Protobuf.swift:27]: SVG is image type but also a vector type

**Review by KaterinaWire — APPROVED** [review]

File: Tests/Source/Model/MockDataModel/MockModelObjectContextFactory.m [hunk 2]
```diff
@@ -87,7 +87,7 @@ + (NSEntityDescription *)createMockEntity2
     [testUUIDDataAttribute setName:@"testUUID_data"];
     [testUUIDDataAttribute setAttributeType:NSBinaryDataAttributeType];
     [testUUIDDataAttribute setOptional:YES];
-    [testUUIDDataAttribute setIndexed:YES];
+//    [testUUIDDataAttribute setIndexed:YES];
```
Comment (billypchan) [inline Tests/Source/Model/MockDataModel/MockModelObjectContextFactory.m:90]:
```suggestion
    [testUUIDDataAttribute setIndexed:YES];
```

File: Tests/Source/Model/MockDataModel/MockModelObjectContextFactory.m [hunk 3]
```diff
@@ -158,7 +158,7 @@ + (NSEntityDescription *)createMockEntityDescription
...
```
Comment (billypchan) [inline Tests/Source/Model/MockDataModel/MockModelObjectContextFactory.m:161]:
```suggestion
    [testUUIDDataAttribute setIndexed:YES];
```

File: Tests/Source/Model/MockDataModel/MockModelObjectContextFactory.m [hunk 4]
```diff
@@ -146,7 +146,7 @@ + (NSEntityDescription *)createMockEntityDescription
     [remoteIdentifierDataAttribute setName:@"remoteIdentifier_data"];
     [remoteIdentifierDataAttribute setAttributeType:NSBinaryDataAttributeType];
     [remoteIdentifierDataAttribute setOptional:YES];
-    [remoteIdentifierDataAttribute setIndexed:YES];
+//    [remoteIdentifierDataAttribute setIndexed:YES];
```
Comment (billypchan) [inline Tests/Source/Model/MockDataModel/MockModelObjectContextFactory.m:149]:
```suggestion
    [remoteIdentifierDataAttribute setIndexed:YES];
```

**Review by johnxnguyen — APPROVED** [review]

**Comment by billypchan** [comment]
@zenkins release major

**Pull request merged**